docs/user-guide/ui.md

```diff
@@ -17,12 +17,17 @@ Dashboard also provides information on the state of Kubernetes resources in your
 
 ## Deploying the Dashboard UI
 
-The Dashboard UI is not deployed by default. To deploy it, run the following command:
+The Dashboard UI is not deployed by default. To deploy it on amd64 servers, run the following command:
 
 ```
 kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
 ```
 
+When deploying to platforms other than amd64 consider execute this command instead. This example replaces the default amd64 docker image for the arm one on the fly:
+
+```
+curl -sSL 'https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml?raw=true' | sed 's/amd64/arm/g' | kubectl create -f -
```

Comment:
I don't think the ?raw=true query parameter is necessary for a request to rawgit.com